.github/workflows/main.yaml

```
@@ -17,22 +17,26 @@ jobs:
   extract-config:
     runs-on: ubuntu-latest
     outputs:
+      flutter_version: ${{ steps.extract.outputs.flutter_version }}
       min_coverage: ${{ steps.extract.outputs.min_coverage }}
       coverage_excludes: ${{ steps.extract.outputs.coverage_excludes }}
     steps:
       - uses: actions/checkout@v4
       - id: extract
         name: Extract configuration values
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:8:44: Double quote to prevent globbing and word splitting

(shellcheck)